capa/features/__init__.py

```diff
@@ -125,6 +138,13 @@ class String(Feature):
     def __init__(self, value, description=None):
         super(String, self).__init__(value, description=description)
 
+    def __str__(self):
+        return 'string("%s")' % self.get_value_str()
```

Comment:
quote the escaped string when displaying for readability.

helps with strings like `"   this string starts with spaces"`.